### DISCUSSION TITLE
Further Clarification on what difficulty values are for Normal/Mythic

### DISCUSSION THREAD

**Discussion by Krateozshael** [chromiecraft/chromiecraft#7366] [PTR Feedback]
With the recent "announcment" of current ptr BT tuning being supposed to be for normal mode, people (including me) felt confused/flabbergasted. In Benchs PTR report it was stated, that those values provided from the PTR are intended to be for mythic, so the difficulty felt mostly okayish for its intended mode.
For Normal tho, the difficulty and values have to be cut down IMMENSLY to serve its purpose.

Still make sure that mythic wont feel overly difficult, so people/guilds are actually playing the mode.

Surely there will be a proper feedback from raidtesting comming, but this posts intention is supposed to make aware of the topic and the thoughts of others.

Please comment for other advises/suggestions

- Shael

**Comment by Nyeriah** [comment]
Going forward we will centralize our tuning changes in the mod-zone-difficulty module we've been using to tune damage and spells. So for the Black Temple values please refer to this file: 

https://github.com/azerothcore/mod-zone-difficulty/blob/master/data/sql/db-world/zone_difficulty_mythicmode_creatureoverrides_bt.sql

Also this: https://github.com/azerothcore/mod-zone-difficulty/blob/master/data/sql/db-world/zone_difficulty_info_content.sql
list of exempt spells here: https://github.com/azerothcore/mod-zone-difficulty/blob/master/data/sql/db-world/zone_difficulty_spelloverrides.sql

mythic trash has a blanket 2x health buff across the board
normal trash has no hp buff
